### PR TITLE
Fea:To handle  500 HTTP responses

### PR DESCRIPTION
**Pull request by shivajihmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-4680

### Change description ###

To handle  500 HTTP responses

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/config/SecurityEndpointFilter.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+
+
+@Configuration
+public class SecurityEndpointFilter extends OncePerRequestFilter {
+
+    public static final String AUTHORIZATION = "Authorization";
+    public static final String SERVICE_AUTHORIZATION2 = "ServiceAuthorization";
+
```
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/config/SecurityEndpointFilter.java:24]: You can remove these unused static variables

Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/config/SecurityEndpointFilter.java:24]: You can remove these unused static variables

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/config/SecurityEndpointFilter.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @Autowired
+    IdamApi idamApi;
```
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/config/SecurityEndpointFilter.java:26]: Can you remove this please as it is not used?

Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/config/SecurityEndpointFilter.java:26]: Can you remove this please as it is not used?

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

**Review by AdilOoze — APPROVED** [review]

**Pull request merged**